.travis.yml

```diff
@@ -65,10 +60,9 @@ notifications:
   email: false
 
 before_install:
-  - if [[ $EXECUTE_TEST_COVERALLS != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
+  - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
   - travis_retry composer self-update
-  - if [[ $EXECUTE_TEST_COVERALLS == 'true' ]]; then composer require --dev --no-update satooshi/php-coveralls ; fi
-  - composer require --no-update "phpunit/phpunit:$PHPUNIT_VERSION"
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update satooshi/php-coveralls ; fi
```

Comment:
`--no-update` make it not install anything php-coveralls bin, change with include $COMPOSER_ARGS and remove the `--no-update` like the following :

```yaml
- if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev satooshi/php-coveralls $COMPOSER_ARGS; fi
```